demo/utilities/router/route-loader.js

```diff
@@ -21,6 +17,13 @@ registerRoutes(
 		},
 		peopleRouteLoader,
 		placesRouteLoader,
+		{
+			pattern: '*',
+			view: () => html`<h1>Not Found</h1>`,
+		}
 	],
-	{ basePath: '/example' }
+	{
+		basePath: '/demo/utilities/router',
+		enableRouteOrderFix: true
```

Comment:
It's unfortunate that new uses always need to enable this. I'm tempted to create a flag that does the inverse (disables the fix), switch all existing places to set that, and then change the behaviour so the fix is applied by default.

Reply:
That would make sense to me